.github/workflows/sidekick.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Sidekick
+on: [push, pull_request, merge_group]
+permissions:
+  contents: read
+jobs:
+  test:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v5
+      - uses: actions/setup-go@v6
+        with:
+          go-version-file: "go.mod"
+      - name: Display Go version
+        run: go version
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - name: Display Rust version
+        run: cargo --version
```

Comment:
optional: this is the version of `cargo` (the package manager) not of Rust (the compiler), you need something like:

https://github.com/googleapis/google-cloud-rust/blob/efbcf8baee844946791db8a0012d585b5db63817/.github/workflows/sdk.yaml#L49-L52